tests/intfstat_test.py

```diff
@@ -110,17 +110,9 @@ def test_clear(self):
         assert result.exit_code == 0
         result = runner.invoke(show.cli.commands["interfaces"].commands["counters"].commands["rif"], [])
         print(result.stdout)
-        expected = ["Ethernet20        0  0.00 B/s    0.00/s         0        0  0.00 B/s    0.00/s         0",
```

Comment:
Can you please include the new output in the test rather than just removing the expected here? I believe it's required to validate the output format